src/main/java/org/apache/sling/scriptingbundle/plugin/processor/Constants.java

```diff
@@ -53,6 +53,10 @@ private Constants() {}
     public static final String BND_SCRIPT_ENGINE_MAPPINGS = "scriptEngineMappings";
     public static final String BND_SEARCH_PATHS = "searchPaths";
 
+    public static final String VAULT_CONTEXT_XML = ".content.xml";
+    public static final String SLING_RESOURCE_SUPER_TYPE = "sling:resourceSuperType";
+    public static final String SLING_REQUIRED_RESOURCE_TYPES = "sling:requiredResourceTypes";
```

Comment:
This requires some documentation as this is not a standard attribute.

Reply:
I will document the usage of this property in a usage example where we document how to use the BND plugin from this module with a FileVault content package.